src/App.js

```diff
@@ -1,8 +1,81 @@
+import React, { useState, useCallback, useRef } from "react";
+import "./style.css";
+import styled from "styled-components";
+import DoneListBox from "./component/DoneListBox";
+import InputBox from "./component/InputBox";
+import TodoListBox from "./component/TodoListBox";
+
+const Wrapper = styled.div`
+  font-family: "EarlyFontDiary";
+  height: 100%;
+  padding: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: whitesmoke;
+  width: 360px;
+  padding: 20px;
+  border-radius: 20px;
+`;
+
 function App() {
+  const [todos, setTodos] = useState([
+    { id: 1, text: "yayaya", checked: true },
+    { id: 2, text: "yayaya", checked: false },
+  ]);
+
+  const doings = todos.filter((todo) => todo.checked === false);
+  const dones = todos.filter((todo) => todo.checked === true);
```

Comment:
```suggestion
  const doings = todos.filter((todo) => !todo.checked);
  const dones = todos.filter((todo) => todo.checked);
```

이렇게 더 간결하게 표현할 수 있을 것 같네요~